python/example_code/aurora/scenario_get_started_aurora.py

```diff
@@ -0,0 +1,252 @@
+        :param parameter_group_name: The name given to the newly created parameter group.
+        :return: The newly created parameter group.
+        """
+        print(f"Checking for an existing cluster parameter group named {parameter_group_name}.")
+        parameter_group = self.aurora_wrapper.get_parameter_group(parameter_group_name)
+        if parameter_group is None:
+            print(f"Getting available database engine versions for {db_engine}.")
+            engine_versions = self.aurora_wrapper.get_engine_versions(db_engine)
+            families = list({ver['DBParameterGroupFamily'] for ver in engine_versions})
+            family_index = q.choose("Which family do you want to use? ", families)
+            print(f"Creating a cluster parameter group.")
+            self.aurora_wrapper.create_parameter_group(
+                parameter_group_name, families[family_index], 'Example parameter group.')
+            parameter_group = self.aurora_wrapper.get_parameter_group(parameter_group_name)
+        print(f"Parameter group {parameter_group['DBClusterParameterGroupName']}:")
+        pp(parameter_group)
+        print('-'*88)
+        return parameter_group
+
+    def set_user_parameters(self, parameter_group_name):
+        """
+        Shows how to get the parameters contained in a custom parameter group and
+        update some of the parameter values in the group.
+
+        :param parameter_group_name: The name of the parameter group to query and modify.
+        """
+        print("Let's set some parameter values in your parameter group.")
+        auto_inc_parameters = self.aurora_wrapper.get_parameters(
+            parameter_group_name, name_prefix='auto_increment')
+        update_params = []
+        for auto_inc in auto_inc_parameters:
+            if auto_inc['IsModifiable'] and auto_inc['DataType'] == 'integer':
+                print(f"The {auto_inc['ParameterName']} parameter is described as:")
+                print(f"\t{auto_inc['Description']}")
+                param_range = auto_inc['AllowedValues'].split('-')
+                auto_inc['ParameterValue'] = str(q.ask(
+                    f"Enter a value between {param_range[0]} and {param_range[1]}: ",
+                    q.is_int, q.in_range(int(param_range[0]), int(param_range[1]))))
+                update_params.append(auto_inc)
+        self.aurora_wrapper.update_parameters(parameter_group_name, update_params)
+        print("You can get a list of parameters you've set by specifying a source of 'user'.")
+        user_parameters = self.aurora_wrapper.get_parameters(parameter_group_name, source='user')
+        pp(user_parameters)
+        print('-'*88)
+
+    def create_cluster(self, cluster_name, db_engine, db_name, parameter_group):
+        """
+        Shows how to create an Aurora cluster that contains a database of a specified
+        type and is configured to use a custom cluster parameter group.
```

Comment:
Shows how to create an Aurora DB cluster that contains a database of a specified
        type. The database is also configured to use a custom DB cluster parameter group.

Reply:
fixed